Content.Server/Fluids/EntitySystems/DrainSystem.cs

```diff
@@ -40,6 +142,12 @@ public override void Update(float frameTime)
                 if (drainSolution is null)
                     continue;
 
+                if (drainSolution.AvailableVolume <= 0)
+                {
+                    _ambientSoundSystem.SetAmbience(drain.Owner, false);
+                    continue;
+                }
+
                 // Remove a bit from the buffer
                 _solutionSystem.SplitSolution(drain.Owner, drainSolution, (drain.UnitsDestroyedPerSecond * drain.DrainFrequency));
```

Comment:
Is having the drain stop at full capacity engaging though, seems like it'd be better as an event or something as it's very easy to do so.